composer.json

```diff
@@ -13,7 +13,7 @@
 		"michelf/php-markdown": "1.5.0",
 		"slim/slim": "^3.0",
 		"pimple/pimple": "^3.0",
-		"illuminate/database": "4.1.*",
+		"illuminate/database": "5.6.*",
```

Comment:
Tagging @NateWr and @kaschioudi in case they're aware of any ramifications we should be aware of.

Reply:
My understanding is that Laravel 5.6 requires PHP 7+. I can't seem to find info specifically on the Illuminate database part of it, but I would guess if Laravel made the jump that it's DB layer would have as well...

Reply:
Yeah, it looks like it expects ^PHP 7.1.3: https://packagist.org/packages/illuminate/database

Reply:
Is this a required bump, @nibou230? We'd like to continue supporting all actively maintained releases of PHP (currently 5.6 or later) and 7.1.3+ is quite an increase.

Reply:
I'm affraid it's required to support SQL server. We tested 4.2.17 and the bug still remains.

You can see the problematic section at the line 56 of the file "https://github.com/illuminate/database/blob/4.2/Connectors/SqlServerConnector.php".

```
return "sqlsrv:Server={$host}{$port}{$dbName}";
```

When the host already contains the "sqlsrv:Server", the produced result looks like : "sqlsrv:Server=sqlsrv:Server" which is not expected by the connector...

Maybe there is an in between version that can fix it while still supporting PHP 5? Else, the only way to still support all versions of PHP would be to ask for a version 4.2.18 that will look at this specific exception and fix it.

Reply:
@nibou230 I suggest bumping to `v5.4.*` which requires `php: >=5.6.4` and I believe corrects the problem you are having (https://github.com/illuminate/database/blob/5.4/Connectors/SqlServerConnector.php#L41-L53)

Reply:
Thanks for the info @kaschioudi, I'll change the version for `v5.4.*` instead.